nequip/data/AtomicData.py

```diff
@@ -249,6 +250,49 @@ def from_ase(cls, atoms, r_max, **kwargs):
             **add_fields,
         )
 
+    def to_ase(self) -> Union[List[ase.Atoms], ase.Atoms]:
+        """Build a (list of) ``ase.Atoms`` object(s) from an ``AtomicData`` object.
+
+        For each unique batch number provided in ``AtomicDataDict.BATCH_KEY``,
+        an ``ase.Atoms`` object is created. If ``AtomicDataDict.BATCH_KEY`` does not
+        exist in self, a single ``ase.Atoms`` object is created.
+
+        Returns:
+            A list of ``ase.Atoms`` objects if ``AtomicDataDict.BATCH_KEY`` is in self
+            and is not None. Otherwise, a single ``ase.Atoms`` object is returned.
+        """
+        positions = self.pos
+        atomic_nums = self.atomic_numbers
+        pbc = getattr(self, AtomicDataDict.PBC_KEY, None)
+        cell = getattr(self, AtomicDataDict.CELL_KEY, None)
+        batch = getattr(self, AtomicDataDict.BATCH_KEY, None)
+
+        if cell is not None:
+            cell = cell.view(-1, 3, 3)
+        if pbc is not None:
+            pbc = pbc.view(-1, 3)
+
+        if batch is not None:
+            unique_batches = range(batch.max() + 1)
+            batch_atoms = []
+            for batch_idx in unique_batches:
```

Comment:
```suggestion
            batch_atoms = []
            for batch_idx in range(batch.max() + 1):
```
no reason to keep unused variable with a name